### PR TITLE
Adding LinkedIn logo to footer

### PR DESCRIPTION
**Pull request by ihsekat**
Adding a LinkedIn logo to the footer next to the FB, Twitter, and G+ icons

### CODE REVIEW

**Comment by ihsekat** [comment]
@stewsmith I think the code is good, could you review? Thanks!

File: website-guts/assets/css/modules/footer.scss [hunk 1]
```diff
@@ -65,6 +65,9 @@ footer {
...
     }
+    .linkedin {
+        background: url('https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/seo-pages/linkedin-icon.png');
```
Comment (stewsmith) [inline website-guts/assets/css/modules/footer.scss]: Just remove the `https:` from these and it looks like we're good.

File: website-guts/assets/css/modules/footer.scss [hunk 2]
```diff
@@ -90,6 +93,9 @@ footer {
       .facebook {
         background: url('//d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/grey-footer/facebook.svg');
       }
+      .linkedin {
+        background: url('https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/seo-pages/linkedin-icon.png');
```
Comment (stewsmith) [inline website-guts/assets/css/modules/footer.scss]: Same as above

File: website-guts/assets/css/om/modules/footer.scss [hunk 3]
```diff
@@ -76,6 +76,9 @@ footer {
...
     }
+    .linkedin {
+        background: url('https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/seo-pages/linkedin-icon.png');
```
Comment (stewsmith) [inline website-guts/assets/css/om/modules/footer.scss]: same as above

File: website-guts/assets/css/om/modules/footer.scss [hunk 4]
```diff
@@ -101,6 +104,9 @@ footer {
       .google-plus {
         background: url('//d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/grey-footer/google.svg');
       }
+      .linkedin {
+        background: url('https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/seo-pages/linkedin-icon.png');
```
Comment (stewsmith) [inline website-guts/assets/css/om/modules/footer.scss]: same as above.

**Comment by stewsmith** [comment]
Logo isn't showing up on the [om/free-trial page](http://optimizely-marketing-site-staging.s3-website-us-east-1.amazonaws.com/tyoung/wrapper/om/free-trial/). You need to add a `li` to `marketing-website/website-guts/templates/om/layouts/wrapper.hbs`

File: website-guts/assets/css/modules/footer.scss (hunk 1 above)
Comment (ihsekat) [inline website-guts/assets/css/modules/footer.scss]: Hey @stewsmith, why would I need to remove the https?

**Comment by stewsmith** [comment]
@ihsekat Please ignore the hound comments. It's something that we're playing with.

File: website-guts/assets/css/modules/footer.scss (hunk 1 above)
Comment (stewsmith) [inline website-guts/assets/css/modules/footer.scss]: @ihsekat For [protocol agnostic urls](http://www.paulirish.com/2010/the-protocol-relative-url/).  I'm just now seeing the update on that article and I'm looking into it now.

**Comment by ihsekat** [comment]
@stewsmith Ok, I've made all the updates if you can take another look.

**Comment by stewsmith** [comment]
LGTM :100:

**Comment by ihsekat** [comment]
:+1:

**Pull request merged**